### PR TITLE
Add issue triaging document

### PR DESCRIPTION
**Pull request by gaiksaya**
### Description
Adding information and instructions to follow for community triage meetings that we will be starting shortly. Thank you to opensearch-security repo [triage doc](https://github.com/opensearch-project/security/blob/main/TRIAGING.md) for reference.

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: TRIAGING.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    1. [opensearch-devops](https://github.com/opensearch-project/opensearch-devops)
+4. Member Requests: Opportunity for any meeting member to ask for consideration of an issue or pull request.
+5. Untriaged Items: Review any [issues](https://github.com/issues?q=is%3Aissue+label%3Auntriaged+repo%3Aopensearch-project%2Fopensearch-build+repo%3Aopensearch-project%2Fhelm-charts+repo%3Aopensearch-project%2Fansible-playbook+repo%3Aopensearch-project%2Fopensearch-ci+repo%3Aopensearch-project%2Fopensearch-cluster-cdk+repo%3Aopensearch-project%2Fterraform-provider-opensearch+repo%3Aopensearch-project%2Fopensearch-devops+repo%3Aopensearch-project%2Fopensearch-build-libraries+is%3Aopen+) that might have had the 'untriaged' label removed but require additional triage discussion.
+6. Open Discussion: Allow for members of the meeting to surface any topics without issues filed or pull request created.
+
+
+There is no specific ordering within each category.
+
+If you have an issue you would like to discuss but do not have the ability to attend the entire meeting please attend when is best for you and signal that you have an issue to discuss when you arrive.
```
Comment (peterzhuamazon) [inline TRIAGING.md:43]:
```suggestion
If you have an issue you would like to discuss but do not have the ability to attend the entire meeting, please attend when is best for you and signal that you have an issue to discuss when you arrive.
```

File: TRIAGING.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+### Do I need to have already contributed to the project to attend a triage meeting?
```
Comment (peterzhuamazon) [inline TRIAGING.md:45]:
```suggestion
### Is prior contribution to the project required to attend a triage meeting?
```

File: TRIAGING.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+No, all are welcome and encouraged to attend. Attending the Backlog & Triage meetings is a great way for a new contributor to learn about the project as well as explore different avenues of contribution.
+
+### What if I have an issue that is almost a duplicate, should I open a new one to be triaged?
```
Comment (peterzhuamazon) [inline TRIAGING.md:49]:
```suggestion
### What if my issue is almost a duplicate? Should I open a new one?
```

File: TRIAGING.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
+
+The maintainers of the OpenSearch-build Repo seek to promote an inclusive and engaged community of contributors. In order to facilitate this, bi-weekly triage meetings are open-to-all and attendance is encouraged for anyone who hopes to contribute, discuss an issue, or learn more about the project. To learn more about contributing to the OpenSearch-build Repo visit the [Contributing](./CONTRIBUTING.md) documentation.
+
+### Do I need to attend for my issue to be addressed/triaged?
+
+Attendance is not required for your issue to be triaged or addressed. All new issues are triaged bi-weekly.
+
+### What happens if my issue does not get covered this time?
```
Comment (peterzhuamazon) [inline TRIAGING.md:9]:
```suggestion
### What happens if my issue does not get covered this week?
```

File: TRIAGING.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
+
+The maintainers of the OpenSearch-build Repo seek to promote an inclusive and engaged community of contributors. In order to facilitate this, bi-weekly triage meetings are open-to-all and attendance is encouraged for anyone who hopes to contribute, discuss an issue, or learn more about the project. To learn more about contributing to the OpenSearch-build Repo visit the [Contributing](./CONTRIBUTING.md) documentation.
+
+### Do I need to attend for my issue to be addressed/triaged?
```
Comment (peterzhuamazon) [inline TRIAGING.md:5]:
```suggestion
### Do I need to attend the meeting for my issue to be addressed/triaged?
```

File: TRIAGING.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+| Label | When applied | Meaning |
+| ----- | ------------ | ------- |
+| Untriaged | When issues are created or re-opened. | Issues labeled as 'Untriaged' require the attention of the repository maintainers and may need to be prioritized for quicker resolution. |
+| Help Wanted | Anytime. | Issues marked as 'Help Wanted' signal that they are actionable and not the current focus of the project maintainers. Community contributions are especially encouraged for these issues. |
+| Good First Issue | Anytime. | Issues labeled as 'Good First Issue' are small in scope and can be resolved with a single pull request. These are recommended starting points for newcomers looking to make their first contributions. |
+
+### Who should I contact if I have further questions?
+
+You can always file an [issue](https://github.com/opensearch-project/opensearch-build/issues/new/choose) for any question you have about the project. Alternatively, you can use [opensearch slack](https://opensearch.org/slack.html) channels named #infra #devops to ask any queries.
```
Comment (peterzhuamazon) [inline TRIAGING.md:69]:
```suggestion
You can always file an [issue](https://github.com/opensearch-project/opensearch-build/issues/new/choose) for any question you have about the project. Alternatively, you can use [opensearch slack](https://opensearch.org/slack.html) channels such as #infra #devops to ask any questions.
```

**Review by peterzhuamazon — COMMENTED** [review]
Hi @gaiksaya 

Add some comments.

Thanks.

File: TRIAGING.md (hunk 4 above)
Comment (gaiksaya) [reply TRIAGING.md:9]: Keeping it time as this is bi-weekly

File: TRIAGING.md (hunk 3 above)
Comment (gaiksaya) [reply TRIAGING.md:49]: Keeping it elaborate for now. We can revisit later.

**Review by peterzhuamazon — APPROVED** [review]

**Review by getsaurabh02 — APPROVED** [review]

**Pull request merged**